js/viz/docs/docSankey.js

```diff
@@ -0,0 +1,484 @@
+            * @type number
+            * @default 0.5
+            */
+            opacity: 0.5,
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.color
+            * @type number
+            * @default undefined
+            */
+            color: undefined,
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.border
+            * @type object
+            */
+            border: {
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.visible
+                * @type boolean
+                * @default undefined
+                */
+                visible: undefined,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.width
+                * @type number
+                * @default undefined
+                */
+                width: undefined,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.color
+                * @type string
+                * @default undefined
+                */
+                color: undefined
+            },
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.hatching
+            * @type object
+            */
+            hatching: {
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.direction
+                * @type Enums.HatchingDirection
+                * @default 'right'
+                */
+                direction: "right",
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.opacity
+                * @type number
+                * @default 0.75
+                */
+                opacity: 0.75,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.step
+                * @type number
+                * @default 6
+                */
+                step: 6,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.width
+                * @type number
+                * @default 2
+                */
+                width: 2,
+            }
+        },
+    },
+    /**
+    * @name dxSankeyOptions.onItemClick
+    * @extends Action
+    * @type function|string
+    * @type_function_param1 e:object
+    * @type_function_param1_field1 jQueryEvent:jQuery.Event:deprecated(event)
+    * @type_function_param1_field2 event:event
+    * @type_function_param1_field3 item:dxSankeyNodeItem|dxSankeyLinkItem
+    * @notUsedInTheme
+    * @action
+    */
+    onItemClick: function() { },
+    /**
+    * @name dxSankeyOptions.onHoverChanged
+    
+    * @extends Action
+    * @type function
+    * @type_function_param1 e:object
+    * @type_function_param1_field1 item:dxSankeyNodeItem|dxSankeyLinkItem
+    * @notUsedInTheme
+    * @action
+    */
+    onHoverChanged: function() { },
+    /**
+    * @name dxSankeyMethods.getAllItems
+    * @return dxSankeyItems
```

Comment:
Just a suggestion: wouldn't it be more convenient to have two methods instead - one that gets all nodes and the other that gets all links?